samples/openapi3/client/petstore/python-experimental/test/test_fruit.py

```diff
@@ -189,5 +189,24 @@ def testFruit(self):
             fruit._additional_properties_model_instances, []
         )
 
+    def testFruitNullValue(self):
+        # Since 'apple' is nullable, validate we can create an apple with the 'null' value.
+        apple = petstore_api.Apple(None)
+        self.assertIsNone(apple)
+
+        # But 'banana' is not nullable.
+        banana = petstore_api.Banana(None)
```

Comment:
@sebastien-rosset 
please take a look at a branch that I made [here](https://github.com/spacether/openapi-generator/tree/all-of-additional-properties_jblack) which updates `banana = petstore_api.Banana(None)` to raise an exception and has this test assert that the exception is raised.
Which do you prefer to get this working?
- merging all-of-additional-properties_jblack into this PR or
- having me build a separate PR which implements _nullable?

If we make a separate _nullable PR, then I would prefer that we
- land the _nullable PR before this one if this contains _nullable code or
- remove the _nullable code from this one and land them separately with no requirement on which lands first

Reply:
Merged.

Reply:
Thank you for the merge!